modules/odc_con_customizing-a-developer-catalog-or-its-sub-catalogs.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_customizing-a-developer-catalog-or-its-sub-catalogs_{context}"]
+= Developer catalog and sub-catalog customization
+
+As a cluster administrator, you have the ability to organize and manage the Developer catalog or its sub-catalogs by enabling or disabling the sub-catalog types or disabling the entire developer catalog. In some situations, customers do not want their users to have access to the Developer catalog or one or more of its sub-catalogs.
+
+`developerCatalog: types` includes the following parameters that you must define in a snippet to use them in the YAML view:
```

Comment:
I'm not sure  `developerCatalog: types` is the preferred way to format these structures - I'm not sure what is, but maybe:

The `developerCatalog.types` object includes the following properties

Reply:
I am a little hesitant to change it to what you have suggested because the SME @debsmita1 has provided the information for this term directly. @debsmita1 - can you let us know if changing this term slightly is fine or not?

Reply:
`developerCatalog.types` should be fine

Reply:
Fixed.